lib/generators/react_on_rails/install_generator.rb

```diff
@@ -97,6 +98,28 @@ def add_post_install_message
         GeneratorMessages.add_info(GeneratorMessages.helpful_message_after_installation)
       end
 
+      def ensure_shakapacker_installed
+        return if shakapacker_installed?
+
+        GeneratorMessages.add_info("Shakapacker not detected. Installing Shakapacker...")
+
+        result = system("rails shakapacker:install")
+        unless result
+          GeneratorMessages.add_error("Failed to install Shakapacker automatically. " \
+                                      "Please run 'rails shakapacker:install' manually.")
+          return
+        end
+
+        GeneratorMessages.add_info("Shakapacker installed successfully!")
```

Comment:
After ShakaPacker is installed, package.json should get updated with the package manager.